### PR TITLE
Add compute synth

### PR DESCRIPTION
**Pull request by chingor13**
Adds compute library synth file which uses the discogapic generator.

Added to synthtool in https://github.com/googleapis/synthtool/releases/tag/synthtool-0.10.0

### CODE REVIEW

**Comment by andreamlin** [comment]
Has this been tested?

**Comment by chingor13** [comment]
These are only really testable by cd'ing to the directory and running `python3 synth.py` (assuming [synthtool](https://github.com/googleapis/synthtool) is already installed). This has been tested locally and there is a documentation change pending which will appear tomorrow assuming this is merged.

File: google-cloud-clients/google-cloud-compute/synth.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""This script is used to synthesize generated parts of this library."""
+
+import synthtool as s
+import synthtool.gcp as gcp
+
+gapic = gcp.DiscoGAPICGenerator()
+common_templates = gcp.CommonTemplates()
```
Comment (andreamlin) [inline google-cloud-clients/google-cloud-compute/synth.py]: Is this variable used somewhere?

Comment (chingor13) [reply google-cloud-clients/google-cloud-compute/synth.py]: Good catch, it's not needed. Will remove.

**Pull request merged**